src/main/java/com/questhelper/quests/gertrudescat/GertrudesCat.java

```diff
@@ -0,0 +1,219 @@
+		giveKittenToFluffy.addSubSteps(climbUpLadderStep);
+
+		return conditionalKitten;
+	}
+
+	private ObjectStep goDownLadderStep()
+	{
+		return new ObjectStep(this, ObjectID.LADDER_11795, new WorldPoint(3310, 3509, 1)
+			, "Climb down ladder");
+	}
+
+	private QuestStep getFeedCat()
+	{
+		gertrudesCat2 = getGertrudesCat("Use seasoned Sardine on Gertrudes cat.", null);
+		gertrudesCat2.addIcon(ItemID.SEASONED_SARDINE);
+
+		ObjectStep climbLadder = new ObjectStep(this, ObjectID.LADDER_11794,
+			new WorldPoint(3310, 3509, 0), "Climb ladder");
+
+		ConditionalStep lumberyard = new ConditionalStep(this, climbLadder, seasonedSardine);
+		lumberyard.addStep(isUpstairsLumberyard, gertrudesCat2);
+		gertrudesCat2.addSubSteps(climbLadder);
+
+		return lumberyard;
+	}
+
+	private QuestStep getLumberyard()
+	{
+		gertrudesCat = getGertrudesCat("Use bucket of milk on Gertrudes cat.", new ItemRequirement("Bucket of milk", ItemID.BUCKET_OF_MILK));
+		gertrudesCat.addIcon(ItemID.BUCKET_OF_MILK);
+
+		ObjectStep climbLadder = getClimbLadder();
+
+		ConditionalStep lumberyard = new ConditionalStep(this, climbLadder, seasonedSardine);
+		lumberyard.addStep(isUpstairsLumberyard, gertrudesCat);
+		gertrudesCat.addSubSteps(climbLadder);
+
+		return lumberyard;
+	}
+
+
+	private NpcStep getGertrudesCat(String text, ItemRequirement requirement)
+	{
+		if (requirement == null)
+		{
+			return new NpcStep(this, NpcID.GERTRUDES_CAT_3497,
+				new WorldPoint(3308, 3511, 1), text);
+		}
+		else
+		{
+			return new NpcStep(this, NpcID.GERTRUDES_CAT_3497,
+				new WorldPoint(3308, 3511, 1), text, requirement);
+		}
+	}
+
+	private QuestStep getTalkToChildren()
+	{
+		NpcStep talkToChildren = new NpcStep(this, NpcID.SHILOP,
+			new WorldPoint(3222, 3435, 0), "Talk to Shilop or Wilough.", true, coins);
+		talkToChildren.addAlternateNpcs(NpcID.WILOUGH);
+		talkToChildren.addDialogStep("What will make you tell me?");
+		talkToChildren.addDialogStep("Okay then, I'll pay.");
+		return talkToChildren;
+	}
+
+	private QuestStep getTalkToGertrude()
+	{
+		NpcStep talkToGertrude = new NpcStep(this, NpcID.GERTRUDE,
+			new WorldPoint(3148, 3413, 0), "Talk to Gertrude.");
+		talkToGertrude.addDialogStep("Yes.");
+		return talkToGertrude;
+	}
+
+	public void setupItemRequirements()
+	{
+		bucketOfMilk = new ItemRequirement("Bucket of milk", ItemID.BUCKET_OF_MILK, 1);
+		coins = new ItemRequirement("Coins", ItemID.COINS_995, 100);
+
+		seasonedSardine = new ItemRequirement("Seasoned Sardine", ItemID.SEASONED_SARDINE, 1);
+		seasonedSardine.setTip("Can be created by using a sardine on Doogle leaves(South of Gertrudes House)");
+	}
+
+	private void setupZones()
+	{
+		Zone zone = new Zone(new WorldPoint(3306, 3507, 12), new WorldPoint(3312, 3513, 1));
+
+		isUpstairsLumberyard = new ZoneCondition(zone);
+	}
+
+	private void setupConditions()
+	{
+		hasFluffsKitten = new ItemRequirementCondition(new ItemRequirement("Fluffs Kitten", ItemID.FLUFFS_KITTEN, 1));
```

Comment:
Default quantity is 1, so it's fine to just go `new ItemRequirementCondition(new ItemRequirement("Fluffs Kitten", ItemID.FLUFFS_KITTEN));`

Reply:
`Fluffs kitten` => `Fluffs' kitten`